### ISSUE TITLE
Installation corrupts by missing some sql statements

### ISSUE THREAD

**Issue by dataviper** [YetiForceCompany/YetiForceCRM#2894]
Hi, I'm trying to import leads into 3.0.0 and every time I get this "Sorry! Attempt to access restricted file"

Tracing it back this is the occurence this is happening:

```
04/22/16 15:07:43,304 [14828] ERROR DB - Query Failed: INSERT INTO vtiger_import_locks VALUES(?,?,?,?,?) | Unknown column 'searchcolumn' in 'field list' | SQLSTATE[23000]: Integrity constraint violation: 1048 Column 'tabid' cannot be null(C:\projects\yeti\modules\Import\actions\Lock.php : 35)
04/22/16 15:07:43,304 [14828] ERROR ROOT - Vtiger_Deprecated:checkFileAccessForInclusion(modules//.php) - Sorry! Attempt to access restricted file. realfilepath: 
04/22/16 15:07:43,304 [14828] ERROR System - Sorry! Attempt to access restricted file. => C:\projects\yeti\vtlib\Vtiger\Deprecated.php:334
```

It's clean fresh install without any data whatsoever and I'm trying to migrate from vtiger

**Comment by dataviper** [comment]
Digging even deeper into the issue it seems like the installer has some kind of strange behaviour.

On 3 different systems 3 different parts of the application broke all because some sql tables, row, or entries were missing.

All installations implied a "clean" install by redirecting to the index page after appearantly all setup was done. 

All php configurations were meeting or exceeding the requirements of yetiforce.
So what actually is needed would be some kind of integrity check or repair tool. If you could provide me a sqldump of an successful, clean installation I'm even willing to write one for you

**Comment by dataviper** [comment]
I changes the title since the problem is related to the database.

A fix for this is implemented in pull request #2906 

A check for correct entries is coming in the next days

**Comment by bpabiszczak** [comment]
1. Please check the latest version of the system.
2. Please check this parameter: 
Requirements for data base engine [MySQL] SQL_MODE should not contain STRICT_TRANS_TABLE
